### DISCUSSION TITLE
Removing HTML tag for Vercel

### DISCUSSION THREAD

**Discussion by ceruleane** [vercel/community#1425] [Help] [answered]
### Summary

Hello, I have deployed several repo through Vercel, but I'm stuck with trying to remove the .html extension. Is there a simple way to fix this issue? I tried the .htaccess file but later found that it is not compatible with Vercel.

**Comment by amyegan** [comment, marked as answer]
Hi @tlsgusdn1107! 👋  You can remove the `.html` extension from the URL by adding a `vercel.json` config with [`cleanUrls`](https://vercel.com/docs/concepts/projects/project-configuration#cleanurls) set to `true`

**Reply by ceruleane** [reply]
Awesome. Thank you!!

**Reply by vikuSingh** [reply]
Thanks..working fine. great job.

**Reply by dtblue-universal** [reply]
is there a video that can help me how to use it im very slow.

**Reply by bloodykheeng** [reply]
just create a file in the root of your project and name it  vercel.json 

then inside that file put this content below 

{
  "cleanUrls": true
}

**Reply by tdbpathiraja** [reply]
Thank you very much this help me lot
